Dockerfile.dev.win

```
@@ -0,0 +1,25 @@
+# escape=`
+
+ARG WINDOWS_VERSION
+FROM docker-registry.devs.facilities.rl.ac.uk/bisapps-node:12-servercore${WINDOWS_VERSION}
+
+#Need permissions to add nessary fonts so chromium can launch
+USER ContainerAdministrator
+
+#Get imagemagick, get all link in the binarys folder then regex to find the latest release
```

I've guessed a bit with this one, may need further adjustment
```suggestion
#Get imagemagick, get all links in the binaries folder, then regex to find the latest release
```